torch/csrc/jit/passes/onnx/unpack_quantized_weights.cpp

```diff
@@ -175,6 +175,100 @@ std::vector<Node*> CreateQuantizedWeights(
   return {const_node_1, const_node_2, const_node_3};
 }
 
+std::vector<Node*> CreateQuantizedWeights(
+    std::shared_ptr<Graph>& graph,
+    const at::Tensor& weight,
+    int8_t* data,
+    const std::vector<int64_t>& shapes,
+    const std::vector<int64_t>& strides) {
+  auto qschema = weight.qscheme();
```

Comment:
schema -> scheme for consistency with with PyTorch nomenclature.
Also update all relevant comments and strings below.

Reply:
done